### ISSUE TITLE
Class 'Cors\Error\AppExceptionRenderer' not found - Cakephp 3.7

### ISSUE THREAD

**Issue by reydajp** [ozee31/cakephp-cors#20]
With Cake 3.7, I get the following error :

`Error: Class 'Cors\Error\AppExceptionRenderer' not found [ROOT\vendor\ozee31\cakephp-cors\src\Error\AppExceptionRenderer.php, line 15]`

Any suggestion ?

Thanks !

**Comment by wschwarte** [comment]
I have the same issue.

seems to have to do something with aliasing a class. It works if i change it to this:
```

function get_dynamic_parent() {
    return Configure::read('Error.baseExceptionRenderer');// return what you need
}
//class_alias(get_dynamic_parent(), 'Cors\Error\BaseExceptionRenderer');   //comment

class AppExceptionRenderer extends ExceptionRenderer // changed name of extended class to cakes default
{


```

**Comment by ozee31** [comment]
I have not this error with CakePHP 3.7 but for me, Cakephp no return a json response with exceptions and I don't understand why.

**Comment by ozee31** [comment]
https://github.com/cakephp/cakephp/issues/12834

**Comment by reydajp** [comment]
The problem has been solved with cakephp 3.7.2 version.
